otx/algorithms/classification/adapters/mmcls/models/heads/cls_head.py

```diff
@@ -8,17 +9,25 @@
 
 @HEADS.register_module(force=True)
 class ClsHead(OriginClsHead):
+    """Classification Head for MMOV inference."""
+
     def __init__(self, *args, **kwargs):
         do_squeeze = kwargs.pop("do_squeeze", False)
-        super(ClsHead, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
         self._do_squeeze = do_squeeze
 
+    def forward(self, x):
```

Comment:
It's just a question. Is this function necessary?

Reply:
Yes, without this, `forward` function is inherited from abstract class.

Reply:
I see. but why previous class doesn't have then?